### PR TITLE
Render on form

### PR DESCRIPTION
**Pull request by phuc-nguyen-tran**
- Render YouTube embedded video on form response
- Fix type error on input minutes and seconds (string -> int)
- Parenthesis bug

### CODE REVIEW

File: client/src/components/VideoForm.js [hunk 1]
```diff
@@ -2,154 +2,166 @@ import styles from "./VideoForm.module.css";
...
+        {errors.length > 0 && (
+          <div>
+            {errors.map((error, index) => (
+              <p key={index}>{error}</p>
+            ))}
+          </div>
+        )}
+      </form>
+
+      {noErrors && (
```
Comment (PhuocOng) [inline client/src/components/VideoForm.js:155]: - this is not a good way to show <Iframe> inside VideoForm.js. VideoForm is components only handle form input, iframe is the video. This set up will make future development harder
**- Recommend:** try to set up redux toolkit, or pass state value from VideoForm.js to Home.js file (pass state value from child-parent)

**Review by PhuocOng — CHANGES_REQUESTED** [review]
- Need some modification

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const Home = () => {
   return (
     <div className={styles.home}>
       <VideoForm />
-      <IFrame youtubeID="MwDYOJNlljE" start="2" end="7" time="3" />
```
Comment (PhuocOng) [inline client/src/pages/Home.js:8]: - Need to find other way to keep <Iframe> in Home.js

**Review by PhuocOng — CHANGES_REQUESTED** [review]

File: client/src/pages/Home.js (hunk 2 above)
Comment (phuc-nguyen-tran) [reply client/src/pages/Home.js:8]: Fixed! Check commit below!

**Review by PhuocOng — APPROVED** [review]

**Pull request merged**